servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -58,14 +59,21 @@
  * Allocator to decide where to put a new block, an Evictor to decide where to evict a stale block,
  * a BlockMetadataManager to maintain the status of the tiered storage, and a LockManager to
  * coordinate read/write on the same block.
- * <p>
+ * <p/>
  * This class is thread-safe.
+ * <p/>
+ * Locking hierarchy
+ * <p/>
+ * A read/write lock to ensure eviction is atomic w.r.t. other operations. An eviction may trigger a
```

Comment:
Is this comment block true anymore? It sounds like the old, big eviction lock comment.

Here, the new locking scheme should be clearly described.